common/src/main/java/com/wynntils/features/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -183,6 +184,23 @@ private void renderTexturedCount(PoseStack poseStack, int x, int y, int emeralds
                     .getItemRenderer()
                     .renderGuiItem(poseStack, EmeraldUnits.values()[i].getItemStack(), renderX + 6, renderY + 6);
 
+            if (EmeraldUnits.values()[i].getSymbol().equals("stx")) { // Make stx not look like normal LE
+                McUtils.mc()
```

Comment:
Oh, I were assuming you created a new texture...

Well, this works, but I think a new texture would make more sense. It makes it available to future changes by an artist (whenever we get one to the team ;-)). And it keeps the code simpler.